docs/framework/migration-guide/versions-and-dependencies.md

```diff
@@ -31,7 +31,7 @@ Each version of the .NET Framework contains the common language runtime (CLR), t
 |4.7|4|- Support for the level of TLS support provided by the operating system.<br/> - Ability to configure default message security settings for TLS1.1 or TLS1.2. <br /> - Improved reliability of the <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer>. <br /> - Improved reliability of serialization and deserialization with WCF applications. <br /> - Ability to extend the ASP.NET object cache. <br /> - Support for a touch/stylus stack based on `WM_POINTER` Windows messages instead of the Windows Ink Services Platform (WISP) for WPF applications. <br /> - Use of Window's Print Document Package API for printing in WPF applications.<br /> - Enhanced high DPI and multi-monitor support for Windows Forms applications running on Windows 10 Creators Update. | | ✓  10 Creators Update <br/> <br/> + 10 Anniversary Update <br/> + 8.1 <br/> +7| + 2016 <br/> + 2012 R2 <br/> + 2012 <br/> + 2008 R2 SP1 |Use `Release` DWORD:<br/><br/> - 460798 (Windows 10 Creators Update) <br/> - 460805 (all other OS versions) <br/><br/> (see [instructions](../../../docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md)) |  
-|4.6.2|4|-   Cryptography enhancements, including support for X509 certificates containing FIS 186-3 DSA, support for persisted-key symmetric encryption, <xref:System.Security.Cryptography.Xml.SignedXml> support for SHA-2 hashing, and increased clarity for inputs to ECDiffieHellman key derivation routines.<br />-   For Windows Presentation Foundation (WPF) apps, soft keyboard support and per-monitor DPI.<br />-   ClickOnce support for the TLS 1.1 and TLS 1.2 protocols.<br />-   Support for converting Windows Forms and WPF apps to UWP apps.||✓  10 Anniversary Update<br /><br /> + 8.1<br />+ 7|✓  2016<br /><br /> + 2012 R2<br />+ 2012<br />+ 2008 R2 SP1|Use `Release` DWORD:<br /><br /> -   394802 (Windows 10 Anniversary Update)<br />-   394806 (all other OS versions)<br /><br /> (see [instructions](../../../docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md))|  
+|4.6.2|4|-   Cryptography enhancements, including support for X509 certificates containing FIS 186-3 DSA, support for persisted-key symmetric encryption, <xref:System.Security.Cryptography.Xml.SignedXml> support for SHA-2 hashing, and increased clarity for inputs to ECDiffieHellman key derivation routines.<br />-   For Windows Presentation Foundation (WPF) apps, soft keyboard support and per-monitor DPI.<br />-   ClickOnce support for the TLS 1.1 and TLS 1.2 protocols.<br />-   Support for converting Windows Forms and WPF apps to UWP apps.||✓  10 Anniversary Update<br /><br /> + 10 November Update <br/> + 10 <br/> + 8.1<br />+ 7|✓  2016<br /><br /> + 2012 R2<br />+ 2012<br />+ 2008 R2 SP1|Use `Release` DWORD:<br /><br /> -   394802 (Windows 10 Anniversary Update)<br />-   394806 (all other OS versions)<br /><br /> (see [instructions](../../../docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md))|  
```

Comment:
For Windows Presentation Foundation (WPF) apps, soft keyboard support, and per-monitor DPI. (missing comma)

Reply:
Do we really need a comma there?